yjit/src/stats.rs

```diff
@@ -144,6 +144,8 @@ make_counters! {
 
     gbpp_block_param_modified,
     gbpp_block_handler_not_iseq,
+
+    string_mismatched_encoding,
```

Comment:
The other counters generally have the name of the op first, could rename this to `concat_encoding_mismatch` to be more consistent.

Reply:
Sure, I'll fix that.